webcompat/static/js/lib/bugform.js

```diff
@@ -347,17 +324,70 @@ function BugForm() {
     removeBanner.removeClass('is-hidden');
     uploadWrapper.addClass('is-hidden');
     removeBanner.on('click', _.bind(function() {
-      // clear out the input value or screenshot data
-      this.uploadField.val(this.uploadField.get(0).defaultValue);
-      this.screenshotData = '';
       // remove the preview and hide the banner
       label.css('background', 'none');
       removeBanner.addClass('is-hidden');
       uploadWrapper.removeClass('is-hidden');
       removeBanner.off('click');
+
+      // remove the last embedded image URL
+      // Note: this could fail in weird ways depending on how
+      // the user has edited the descField.
+      this.descField.val(function(idx, value) {
+        return value.replace(/!\[.+\.jpe*g\)$/, '');
```

Comment:
When you click "Remove image upload", it removes the image preview *and* removes the last `![Screenshot descriptions](http://example.com/link/to/image.jpg)` in the bug description. And actually it's got a bug in it! We can also upload gif and bmp (png will always be jpg).